objects/ConsoleExtension/Removable Media Security (secRMM) ConfigMgr console extension.json

```diff
@@ -1,6 +1,6 @@
 {
   "downloadLocation": "https://www.squadratechnologies.com/StaticContent/ProductDownload/secRMM/9.11.0.0/secRMMConfigMgrConsoleExtension.cab",

```

Comment:
Kevin's last comment is key here.  So you should complete this ASAP since folks are blocked on downloading your existing version until you complete this PR. 

We'll also make a change that will check for this in the future so this mistake can't be made.

Reply:
Hi guys, ugh, I did not see this, sorry!
OK, for now, it is possible to use the same location (understanding your comments about previous customer...which should be 0 right now)?
I have a customer waiting on this.
I will change the build process after this one to include the version in the file name so that the location (via the file name) will always change.
Again, sorry for the trouble (I am learning! :-) )
Thanks for all the support!